storage/src/test/java/org/apache/kafka/tiered/storage/TieredStorageTestContext.java

```diff
@@ -238,7 +242,11 @@ public void stop(int brokerId) {
     }
 
     public void start(int brokerId) {
+        boolean allBrokersDead = harness.aliveBrokers().isEmpty();
         harness.startBroker(brokerId);
+        if (allBrokersDead) {
+            reinitClients();
+        }
```

Comment:
Will this cause we init clients twice when 1st start up? 
If we already reinit clients in `bounce` method, why should we reinit clients here?

Reply:
On broker startup, this method won't be invoked. Assume that the user does `stop` and `start` instead of `bounce`, then we may have to reinitialise the clients.

Reply:
OK, I see.